### ISSUE TITLE
md2html could accept an encoding parameter

### ISSUE THREAD

**Issue by 1kastner** [evilstreak/markdown-js#245]
My idea would be, that when you pass a parameter for the encoding, this is put into the html file. Kind of:

> md2html myFile.md --encoding=utf8 > myFile.html

And then utf8-Encoding would be automatically part, like <meta charset="utf-8"/>

**Comment by ashb** [comment]
Sounds like a nice idea (your example got cut off but I think I know what you were going to say - probably you need to escape it so github doesn't sanitize it)

I'll happily merge a pull request that does this cleanly.

**Comment by 1kastner** [comment]
yes, I forgot about that, now you should see it. I will try to find a nice solution.

**Comment by 1kastner** [comment]
I have done this: https://github.com/1kastner/markdown-js
It seems to work nicely for me, it is definitely doing its job, but there is one small problem: To me it looks like all tests are failing now. I am quite new to node-js-programming and I am not sure how this is happening or whether I am missinterpreting something. If you could have a short glance, maybe it could help. Thanks!

**Comment by 1kastner** [comment]
I have provided a pull request. I hope you are happy with what I am
doing with the options-parameter. Whatever you prefer, it can be changed.

Am 11.07.15 um 13:40 schrieb Ash Berlin:

**Comment by 1kastner** [comment]
#246 got declined...

**Comment by tobixen** [comment]
I also need this.  Tried using md2html on a text I had written, and the non-ascii letters came out as complete gibberish.

**Comment by 1kastner** [comment]
Have you tried on of the alternatives listed at https://github.com/evilstreak/markdown-js#notice-unmaintained ?
